src/metadata_tool/tools/generators.clj

```diff
@@ -24,7 +24,12 @@
 (defn gen-clabot-whitelist
   []
   (println (tem/render "clabot-whitelist.ftl"
-                       {:github-ids (sort (mapcat :github-logins (md/people-with-clas)))})))
+                       {:github-ids (sort (mapcat :github-logins (md/people-with-clas)))
+                        :email-domains 
+                          (sort (flatten 
+                            (mapcat :domains 
+                              (filter #(:cla-email-whitelist %) 
```

Comment:
this is a bit better as `(filter :cla-email-whitelist (md/organizations-metadata))`
I'd also consider using `->>`, looks simpler to me but YMMV.  

I don't know what the data structure looks like anymore (I don't have access to the `metadata` repo), so it's hard to figure out, but `flatten` is usually a smell.  `filter`, then `mapcat`, then `flatten` might have a better solution.    Minor syntax improvement, so just a suggestion.

Reply:
TY @tschady !

Re `->>`, I'm trying to use it, but takes a bit of practice I guess, so I'll schedule it for a future code refactoring.

Re. the `flatten`, this is the JSON of the FINOS org in our metadata DB:

```
{
  "metadataVersion"  : "2.1.0",
  "organizationName" : "FINOS",
  "hasCCLA"          : true,
  "domains"          : [ "finos.org" ]
}
```

The flatten is supposed to "extract" the list of `domains` values. Happy to apply a better approach of course!

Reply:
in that case you don't need `flatten` at all, it's not doing anything.  `mapcat` gets you what you need.

You use vscode, right?  Checkout Calva, I hear good things.  Can instantly swap between threaded and unrolled forms.

Reply:
Wow, Calva is fantastic, thank you! Colour-coded parenthesis are super helpful.

Reply:
Note that so-called "Rainbow Parentheses" have a long tradition in the Lisp/Clojure community, and are available in most editors.